legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-bigquery/legend-engine-xt-relationalStore-bigquery-wrapper/checkstyle.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0"?>
```

Comment:
Update copyright header to mention Google.

Reply:
Actually .. Don't think we need this checstyle.xml here. The root checkstyle.xml should suffice